src/components/fields/chips/chips.tsx

```diff
@@ -27,12 +28,19 @@ export const Chips = (props: IGenericFieldProps<IChipsSchema>) => {
 	const [stateValue, setStateValue] = useState<string[]>(value || []);
 	const [showTextarea, setShowTextarea] = useState(false);
 	const [multi, setMulti] = useState(true);
-	const { control, setValue } = useFormContext();
+	const { setValue, unregister } = useFormContext();
 	const { setFieldValidationConfig, removeFieldValidationConfig } = useValidationConfig();
 
 	// =============================================================================
 	// EFFECTS
 	// =============================================================================
+	useEffect(() => {
+		if (!showTextarea) {
+			// at the start, the textarea's default value has to be manually cleared from the form state
+			unregister(getTextareaId());
```

Comment:
just to clarify, is this a temp fix before you make a complete fix in another PR?
I wonder if `unregister` can be used in FieldWrapper or ConditionalRenderer

Reply:
yup this is temporary. I'm intending to run `unregister` for each `idsToDelete` in `ConditionalRenderer`. `FieldWrapper` won't work for all cases as it requires the component to unmount which doesn't happen on fields nested within conditionally rendered fields